### DISCUSSION TITLE
Redirect to hosted_payment_url affiche une erreur "404 Can't retrieve merchand with Payment Order ID"

### DISCUSSION THREAD

**Discussion by Didier73100** [PayGreen/documentation#5] [Q&A] [answered]
J'ai créé un payment order via : https://developers.paygreen.fr/reference/post_create_payment_order.
J'ai une réponse 200 Ok, quand j'ouvre l'url du champ hosted_payment_url dans mon navigateur j'obtiens une  erreur "424 Dependency failed." ou une erreur "404 Can't retrieve merchand with Payment Order ID".
Idem depuis le client PHP et avec Postmann/Insomnia.
J'ai essayé d'inclure le bearer Token a la requete GET pour charger la page mais ça ne change rien.
Auriez vous une idée ?

**Comment by IMP3ter** [comment, marked as answer]
Bonjour Didier,

Pourriez-vous vérifier si une public key est bien générée sur votre compte ?
Vous pouvez y accéder ici : https://app.paygreen.fr/account/tokens

**Reply by Didier73100** [reply]
Merci pour votre aide
